.changeset/proud-carrots-press.md

```diff
@@ -0,0 +1,8 @@
+---
+'@commercetools-docs/gatsby-theme-api-docs': patch
+'@commercetools-docs/gatsby-transformer-raml': patch
+'@commercetools-docs/ramldoc-generator': patch
+'@commercetools-docs/rmf-codegen': major
```

Comment:
```suggestion
'@commercetools-docs/rmf-codegen': minor
```

As otherwise the theme packages are also bumped as major. Unless you want to do that of course.

Reply:
OK, looks like the better compromise.  I intended to have the new package start with a  [major].0.0  but it's just a formality so it'll start at 1.1.0 now.